gdb/testsuite/gdb.rust/simple.exp

```diff
@@ -83,6 +81,30 @@ gdb_test "print z.1" " = 8"
 gdb_test "print simple::ByeBob(0xff, 5)" \
     " = simple::ByeBob \\(255, 5\\)"
 
+gdb_test "print e" " = simple::MoreComplicated::Two\\(73\\)"
+gdb_test "print e2" \
+         " = simple::MoreComplicated::Four\\{this: true, is: 8, a: 109 'm', struct_: 100, variant: 10\\}"
+
+gdb_test_sequence "ptype e" "" \
+    [list " = enum simple::MoreComplicated \\{" \
```

Comment:
I realized later that these `list` calls aren't needed and they can just be a braced list.